int32 task_command_id
int32 work_type_id
int32 number_of_items_picked
WorkPresenceArea work_presence_area
CandidateDischargeLocationAreaList candidate_discharge_location